### ISSUE TITLE
👋🏼 Curious Check In

### ISSUE THREAD

**Issue by shellscape** [hedgepigdaniel/webpack-hmr-client#5]
Hey folks, I noticed that this was forked from my fork and was just checking in out of curiosity to see what you all were using the fork/renamed module for. NPM didn't have any dependents listed, so I figured I'd open an issue here.

**Comment by hedgepigdaniel** [comment]
Hi!

I forked it from https://github.com/webpack-contrib/webpack-hot-client - just to merge outstanding bugfixes for very large bugs. A few other people seemed interested so I renamed it for the purpose of putting it on NPM.

**Comment by shellscape** [comment]
Right on. Thanks for replying! If you've any interest in publishing your changes to `webpack-hot-client` on npm, please let me know. The module is published from my fork of the original webpack-contrib project I authored and I'd welcome PRs. No worries if not, but offer is on the table! Cheers!

**Comment by hedgepigdaniel** [comment]
Interesting. I'm keen to combine efforts.

Why does the webpack-contrib fork exist then? (especially since it gives the impression that the package is published from it).

Do you have write access to the webpack-contrib fork? My tendency would be to merge the PRs there and publish it from there, since that repo is much easier to find, and that's where people report issues.

**Comment by shellscape** [comment]
The short story is that I was webpack core for a year and a half, and published most of the new modules I created from webpack-contrib. I took a leave of absence after some disagreements with the org leadership and returned to find most of my work deprecated. I forked those packages that I had publish rights to, to my personal Github account and updated the package metadata. All of those packages are now published from my repos. I can't speak towards why they still keep that repo active, as they don't maintain it any longer, but they have updated repos such as https://github.com/webpack-contrib/webpack-serve to note that it's moved.

> since that repo is much easier to find

I suppose that depends on where one is coming from. The package does point to my repo on NPM. 

_Edit: Anyone who'd like to maintain it beyond dependency and security updates, I'm happy to give permissions to do so._
